src/Command/Command.php

```diff
@@ -463,12 +431,12 @@ public function getFetchMode(): int
 
     public function getParams(): array
     {
-        return $this->params;
-    }
-
-    public function getPdoStatement(): ?PDOStatement
-    {
-        return $this->pdoStatement;
+        return array_map(
+            static function ($value) {
+                return $value->getValue();
+            },
+            $this->params
+        );
```

Comment:
`return array_map(static fn (mixed $value): mixed => $value->getValue(), $this->params);`